runtime/devnet/src/lib.rs

```diff
@@ -415,6 +417,10 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = ();
 	type PriceForSiblingDelivery = NoPriceForMessageDelivery<ParaId>;
+	// Limit the number of messages and signals a HRML channel can have at most
+	type MaxActiveOutboundChannels = ConstU32<128>;
+	// Limit the number of HRML channels
+	type MaxPageSize = ConstU32<{ 103 * 1024 }>;
```

Comment:
base parachain and sdk:
```
type MaxPageSize = ConstU32<{ 1 << 16 }>;
```

Reply:
Also testnet

Reply:
I see this change was made for the storage reclaim. This is however not changed in the polkadot sdk template. Do you have a reference for why it was changed to this value?

Reply:
@Daanvdplas https://github.com/r0gue-io/pop-node/pull/198#issuecomment-2297673149 this is proposed by @al3mart for the sake of simplicity that fits the use case of pop network at the moment.

Reply:
Oke sounds good